ibis/backends/base/sql/alchemy/client.py

```diff
@@ -329,3 +329,120 @@ def _sqla_table_to_expr(self, table):
     def version(self):
         vstring = '.'.join(map(str, self.con.dialect.server_version_info))
         return parse_version(vstring)
+
+    def insert(
+        self,
+        to_table_name: str,
+        database: Optional[str] = None,
+        data_obj=None,
```

Comment:
can you type this?

is this signature consistent in naming? e.g. is `data_obj` what we call this generally?

Reply:
`data_obj` was earlier intended to send either pandas dataframe or a table expression. Later, we decided to only send the dataframe and if table expression needs to be used another variable with its workflow or some other change as suggested in this [issue](https://github.com/ibis-project/ibis/issues/2778) will be implemented.  
If the variable only uses dataframe we have used `data` as variable name in `.load_data()` function. So, I'll name it the same here.